XLPandXeLaTeX.ltx

```diff
@@ -46,74 +46,69 @@ producing complex documents in beautiful formats. \TeX
 \cite{knuth:1984},\cite{knuth:1986} is one of the well known
 publishing technologies used to meet these needs. Since 2000, \XML
 based technologies such as
-XSL-FO\footnote{\url{https://www.w3.org/TR/xsl11}} or the TeXML
+XSL‑FO\footnote{\url{https://www.w3.org/TR/xsl11}} or the \TeXML
```

Comment:
\TeXML was failing to process.  I've just now learned that we can use
\TeX{}ML and it will come out the way we want.  I'll make those changes.

Reply:
Ok, so `\tex{}` is a macro to style the TeX text?

Reply:
Yes, althouh it is \TeX (followed by a space or "{}").